scripts/base/frameworks/openflow/consts.zeek

```diff
@@ -123,7 +123,7 @@ export {
 	## Return value for a cookie from a flow
 	## which is not added, modified or deleted
 	## from the Zeek openflow framework.
-	const INVALID_COOKIE = 0xffffffffffffffff;
+	const INVALID_COOKIE = 0x7fffffffffffffff;
```

Comment:
Enums have `int`-valued values, not `count`-valued.  The above constant is a `count`, so it overflows on the conversion.  Presumably the value was chosen to for-sure-not-collide with any other chosen value.  For that, max-int seems a better choice than -1.  (I found this because the script-to-C++ compiler generated the constant using its intended type but its actual unsigned value, which led to a C++ compilation-time warning.)

Reply:
I originally missed the connection to enums, but now see there's the `type ethertype: enum` commented out and wonder: are these names actually going to change to enum values later?  And for what reason?

Reply:
Darned if I know.  I scanned and saw the `enum` and missed the fact that it's commented out.  So I think this was just a bug in the C++ translation.  The original value could stand, though it also seems fine to stick with the signed-int-appropriate version, now that it's merged.